src/java/com/pb/mtctm2/abm/ctramp/ExplicitTelecommuteModel.java

```diff
@@ -0,0 +1,146 @@
+package com.pb.mtctm2.abm.ctramp;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Random;
+import com.pb.common.calculator.VariableTable;
+import com.pb.common.model.ModelException;
+import com.pb.mtctm2.abm.accessibilities.AccessibilitiesTable;
+import com.pb.mtctm2.abm.accessibilities.MandatoryAccessibilitiesCalculator;
+
+import org.apache.log4j.Logger;
+
+public class ExplicitTelecommuteModel
+        implements Serializable
+{
+
+    private transient Logger                   logger   = Logger.getLogger(ExplicitTelecommuteModel.class);
+    private transient Logger                   etLogger = Logger.getLogger("et");
```

Comment:
"telecommute" may be a handier reference than "ET". @lmz: do you have a preference for the abbreviation? Some of these will show up in your properties file. Perhaps `tele` rather than `et` if we want to be brief?